### PR TITLE
OSDOCS-5588 RNs for 4.12.9 z-stream release

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.12

Issue:
[OSDOCS-5588](https://issues.redhat.com/browse/OSDOCS-5588)

Link to docs preview:
[Preview](https://57681--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-9)

QE review:
- [ ] QE has approved this change.

Additional information:
Z-stream release notes for 4.12.9.

### CODE REVIEW

**Review by snarayan-redhat — COMMENTED** [review]
Great work!

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -3377,4 +3377,30 @@ $ oc adm release info 4.12.8 --pullspecs
...
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-9"]
+=== RHSA-2023:1409 - {product-title} 4.12.9 bug fix and security update
+
+Issued: 2023-03-27
+
+{product-title} release 4.12.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:1409[RHSA-2023:1409] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:1408[RHSA-2023:1408] advisory.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-12-release-notes.adoc:3387]: Should this be 4.12.9?

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -3377,4 +3377,30 @@ $ oc adm release info 4.12.8 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.12.9 --pullspecs
+----
+
+[id="ocp-4-12-9-bug-fixes"]
+==== Bug fixes
+
+* Previously, validation was not preventing users from installing a GCP cluster into a shared VPC if they didn't enable the technology preview feature gate. Therefore, you could install a cluster into a shared VPC without enabling the technology preview feature gate. This release added a feature gate validation to 4.12 so you must enable `featureSet: TechPreviewNoUpgrade` to install a GCP cluster into a shared VPC. (link:https://issues.redhat.com/browse/OCPBUGS-7469[*OCPBUGS-7469*])
```
Comment (bergerhoffer) [inline release_notes/ocp-4-12-release-notes.adoc:3399]: A few minor things to fix

```suggestion
* Previously, validation was not preventing users from installing a GCP cluster into a shared VPC if they did not enable the Technology Preview feature gate. Therefore, you could install a cluster into a shared VPC without enabling the Technology Preview feature gate. This release added a feature gate validation to 4.12 so you must enable `featureSet: TechPreviewNoUpgrade` to install a GCP cluster into a shared VPC. (link:https://issues.redhat.com/browse/OCPBUGS-7469[*OCPBUGS-7469*])
```

**Review by bergerhoffer — COMMENTED** [review]
A few minor things to fix before merging.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc:3387]: Yes! Thank you!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (cbippley) [reply release_notes/ocp-4-12-release-notes.adoc]: Done!

**Pull request merged**